_workinggroups/visualization.md

```diff
@@ -1,50 +1,54 @@
 ---
 title: "Visualization"
 author: "Riccardo Maria Bianchi"
-layout: default
-redirect_from: /activities/training.html
+layout: plain
+redirect_from: /activities/visualization.html
 ---
 
 # Visualization
 
-The Visualization Working Group (WG) gathers experts, developers, users and all people from the HEP community interested in interactive data visualization and event displays. The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh*).
+The Visualization Working Group (WG) gathers experts, develeopers, users and all people from the HEP community interested in interactive data visualization and event displays. 
+
+The WG has been started in October 2016 and it is convened by Riccardo Maria Bianchi (*University of Pittsburgh, US*, ATLAS Experiment). 
```

Comment:
You should put some contact information in here - which should be a way to contact the convenor (i.e., you!) and a way to contact the group (I suppose there's a mailing list? A few of the groups renamed their mailing lists to remove the "cwp" part, but retain the established contacts).

Reply:
Right, so checking, what I suggest is to rename "hsf-cwp-visualization" -> "hsf-visualization-wg"

Reply:
Sorry, Just saw this one! 
I'm checking that now.

Reply:
OK, done. Added to both Viz and QC.

Reply:
and I renamed the Viz mailing list